### PR TITLE
refactor class MavenWrapperDownloader

### PR DESCRIPTION
**Pull request by robatipoor**

### CODE REVIEW

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -21,97 +21,91 @@
...
     /**
-     * Name of the property which should be used to override the default download url for the wrapper.
+     * Name of the property which should be used to override the default download
+     * url for the wrapper.
      */
     private static final String PROPERTY_NAME_WRAPPER_URL = "wrapperUrl";
 
     public static void main(String args[]) {
         System.out.println("- Downloader started");
+        if (args.length != 1) {
```
Comment (fbricon) [inline .mvn/wrapper/MavenWrapperDownloader.java]: args.length < 1

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 2]
```diff
@@ -21,97 +21,91 @@
...
+            throw new IllegalArgumentException();
```
Comment (fbricon) [inline .mvn/wrapper/MavenWrapperDownloader.java]: Add message to provide feedback on the failure

**Review by fbricon — CHANGES_REQUESTED** [review]

File: .mvn/wrapper/MavenWrapperDownloader.java (hunk 1 above)
Comment (robatipoor) [reply .mvn/wrapper/MavenWrapperDownloader.java]: ok done

**Comment by mosabua** [comment]
This project is essentially frozen for development and new efforts are ongoing in the Apache Maven project itself. We therefore close this PR. Please find more information at https://github.com/takari/maven-wrapper#ongoing-migration-to-apache-maven and participate in the development there.

**Pull request closed without merging**